### PR TITLE
Create 0000-01-02-coco243.md

### PR DESCRIPTION
**Pull request by coco243**
new stpeps

### CODE REVIEW

File: _posts/0000-01-02-coco243.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Let's get on buisness!
```
Comment (coco243) [inline _posts/0000-01-02-coco243.md]: modif

Comment (coco243) [reply _posts/0000-01-02-coco243.md]: it is ok

Comment (coco243) [inline _posts/0000-01-02-coco243.md]: yes

**Review by coco243 — COMMENTED** [review]
ok

**Comment by coco243** [comment]
f

**Comment by coco243** [comment]
Please review so I can go further.

**Review by coco243 — COMMENTED** [review]
a

File: _posts/0000-01-02-coco243.md (hunk 1 above)
Comment (coco243) [inline _posts/0000-01-02-coco243.md]: REVIEW

**Review by coco243 — COMMENTED** [review]
K

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]
qwq

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]
asa

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]
k

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Comment by ghid84** [comment]
k

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]
ok

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]
goody

**Comment by ghid84** [comment]
ok

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

File: _posts/0000-01-02-coco243.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+Use the left arrow to go back!
```
Comment (ghid84) [inline _posts/0000-01-02-coco243.md:6]: h

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Review by ghid84 — APPROVED** [review]

**Pull request merged**